.github/workflows/lambda_review.yml

```diff
@@ -0,0 +1,48 @@
+
+name: Build and push lambda image to staging for this PR (temporary)
```

Comment:
🔥 this before merging. Or maybe just set it not to run, it's kinda handy to have around.

Reply:
I would be in favour of removing this for now and then bringing it back if/when we decide to set up API review apps properly